tests/baselines/reference/asOpEmitParens.js

```diff
@@ -0,0 +1,19 @@
+//// [asOpEmitParens.ts]
+declare var x;
+// Must emit as (x + 1) * 3
+(x + 1 as number) * 3;
```

Comment:
Ah, derp

On Fri, Jul 15, 2016, 5:54 PM Ryan Cavanaugh <notifications@github.com>
wrote:

> In tests/baselines/reference/asOpEmitParens.js
> <https://github.com/Microsoft/TypeScript/pull/9767#discussion_r71057592>:
>
> > @@ -0,0 +1,19 @@
> > +//// [asOpEmitParens.ts]
> > +declare var x;
> > +// Must emit as (x + 1) * 3
> > +(x + 1 as number) * 3;
>
> You're looking at the wrong part of the baseline (this is the original
> .ts). Its corresponding emitted line is 15
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/Microsoft/TypeScript/pull/9767/files/40ca4a06b55d9493a7ed45bd92543a0a14a7ac3a#r71057592>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACzAMnkNN2tSPIQlLgSHEVrLTCD7tddLks5qWCvCgaJpZM4JN2j9>
> .
>